apps/ledger-live-desktop/tests/models/SettingsPage.ts

```diff
@@ -73,4 +77,18 @@ export class SettingsPage {
     await this.themeSelector.click();
     await this.themeChoiceLight.click();
   }
+
+  async enableAndGoToDeveloperTab() {
+    await this.versionRow.click();
+    await this.versionRow.click();
+    await this.versionRow.click();
+    await this.versionRow.click();
+    await this.versionRow.click();
+    await this.versionRow.click();
+    await this.versionRow.click();
+    await this.versionRow.click();
+    await this.versionRow.click();
+    await this.versionRow.click();
+    await this.developerTab.click();
+  }
```

Comment:
![doubt-pout](https://user-images.githubusercontent.com/91890529/222098327-ab1d2620-3013-4015-b787-1e4cd8f60458.gif)

Reply:
surely this can be done with a loop no? 😄